server/server.py

```diff
@@ -24,34 +24,37 @@ def handle(self):
 		try:
 			msg = self.RSAcipher.decrypt(data).decode().split(" ", 1)
 
-			if (msg[0] == '3317BLT5'):
+			if (msg[0] != '3317BLT5'):
+				raise BaseException("Invalid magic number")
```

Comment:
Agreed, the error logging would go in the except. Could make a good improvement issue for later on.